test/routes/src/server/routes/t_launchpad.js

```diff
@@ -78,8 +78,12 @@ describe('The Launchpad API endpoint', () => {
           .reply(200, 'name: test-snap\n');
         const lp_api_url = conf.get('LP_API_URL');
         nock(lp_api_url)
-          .post('/devel/+snaps')
-          .query({ 'ws.op': 'new' })
+          .post('/devel/+snaps', {
+            'ws.op': 'new',
+            git_repository_url: 'https://github.com/anaccount/arepo',
+            processors: ['/+processors/amd64', '/+processors/armhf'],
```

Comment:
Yep, exactly that.  I think it's probably best not to duplicate the expectations.